# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/html/rendering/non-replaced-elements/the-page/iframe-scrolling-attribute.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/html/rendering/non-replaced-elements/the-page/iframe-scrolling-attribute.html.ini
--- a/testing/web-platform/meta/html/rendering/non-replaced-elements/the-page/iframe-scrolling-attribute.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/html/rendering/non-replaced-elements/the-page/iframe-scrolling-attribute.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -2,6 +2,6 @@
   fuzzy:
     if os == "win": maxDifference=0-4;totalPixels=0-51
   expected:
+    if (os == "linux") and not webrender and not debug: ["PASS", "FAIL", "ERROR"]
     if (os == "linux") and not webrender and debug: ["PASS", "FAIL", "ERROR"]
-    if (os == "linux") and not webrender and not debug: ["PASS", "FAIL", "ERROR"]
     if os == "mac": ["PASS", "FAIL"]